Create a document to set up databases for Scalar DB/Scalar DL deployment in AKS (#162)
scalar-labs/scalar-kubernetes: docs/SetupDatabaseAKS.md

> # Set up a database for Scalar DL/Scalar DB deployments in AKS

Reviewer: ```suggestion
# Set up a database for Scalar DB / DL deployments in AKS
```

Author: I have not changed it as of now. I am not sure this can be considered as a redundancy.  I will check further and confirm if needed.

> This guide explains how to set up a database for Scalar DL/Scalar DB deployments in AKS.

Reviewer: ```suggestion
This guide explains how to set up a database for Scalar DB / DL deployments in AKS.
```

## Cosmos DB

> In this section, you will set up a Cosmos DB for Scalar DL/Scalar DB.

Reviewer: ```suggestion
In this section, you will create a Cosmos DB account.
```

Author: Thank you for the suggestion. Updated sentence.

Scalar schema tool abstracts Request Unit of Cosmos DB with RU. So please set the appropriate value.

#### Autoscale

Autoscale provisioned throughput in Azure Cosmos DB allows you to scale the throughput (RU/s) of your database or container automatically and instantly.
The throughput is scaled based on the usage, without impacting the availability, latency, throughput, or performance of the workload.
By default, the scalardl schema tool enables autoscale of RU for all tables: RU is scaled in or out between 10% and 100% of a specified RU depending on a workload.

### Monitor Cosmos DB

By default, monitoring is enabled on Cosmos DB.

More information can be found in [Monitor Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/monitor-cosmos-db) documentation.

## Azure Database for MySQL

Azure Database for MySQL is a fully managed MySQL database provided by Azure. You can create a MySQL database by selecting a database instance and storage according to your configurations. The IOPS is calculated based on the amount of storage you choose.
In this section, you will create an Azure Database for MySQL.

### Recommendations

> You can choose Flexible or Single server deployment for Azure Database for MySQL.

Reviewer: I think it is better to add as a list

Reviewer: Just a suggestion:
I think we need to add a recommendation section to specify the IOPS.
Maybe we can't specify a fixed IOPS but we can share awareness of it.

Author: Thank you. Updated recommendations section.

You can check the [official documentation](https://docs.microsoft.com/en-us/azure/mysql/select-right-deployment-type) to learn more about the differences between Flexible and Single server deployments.

### Steps

You can follow [official documentation](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal) to create an Azure Database for MySQL server for Single server deployment.

> For Flexible server deployment for the same, you can follow this [guide](https://docs.microsoft.com/en-us/azure/mysql/flexible-server/quickstart-create-server-portal).

Reviewer: Sorry, I feel some ambiguity for  `For Flexible server deployment for the same` sentence.

Author: Thank you. Updated the sentence to be more clear.

### Scale Performance

**Scaling storage**:- You can enable the `storage auto grow` feature to automatically increase storage when it reaches a threshold without impacting the workload.

### Monitoring Azure Database for MySQL

By default, Azure Database for MySQL servers has monitoring enabled.
You can monitor database metrics from the Azure portal.
For more information, check the [official documentation](https://docs.microsoft.com/en-us/azure/mysql/concepts-monitoring).

> ## Azure Database for PostgreSQL

Reviewer: All changes in the above section need to be made here as well

Author: Changes made similar to  MySQL section.
